### DISCUSSION TITLE
/traefik-v2-and-external-services-like-home-assistant/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#175] [Announcements]
Traefik v2 and external services like Home Assistant and Blue Iris
        
[https://blog.ktz.me/traefik-v2-and-external-services-like-home-assistant/](https://blog.ktz.me/traefik-v2-and-external-services-like-home-assistant/)

**Comment by inf0maniac** [comment]
thanks!

**Comment by willharwell** [comment]
Thank you for the write-up. It got me working in less than 5 minutes.

**Comment by akavoukis** [comment]
> I lost most of an afternoon because certresolver is not the same as certResolver and so on.

Thanks you saved mine
